terraform/helm/chaos/templates/ingress.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.ingress.enable }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "chaos.fullname" . }}
+  labels:
+    {{- include "chaos.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/tags: {{ .Values.ingress.aws_tags | quote }}
+    {{- if .Values.ingress.loadBalancerSourceRanges }}
+    alb.ingress.kubernetes.io/inbound-cidrs: {{ join "," .Values.ingress.loadBalancerSourceRanges }}
+    {{- end }}
+    external-dns.alpha.kubernetes.io/hostname: {{ .Values.ingress.domain }}
+    {{- if .Values.ingress.acm_certificate }}
+    alb.ingress.kubernetes.io/certificate-arn: {{ .Values.ingress.acm_certificate }}
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
+    alb.ingress.kubernetes.io/ssl-redirect: '443'
+    {{- end }}
+spec:
+  rules:
+  - host: {{ .Values.ingress.domain }}
+    http:
+      paths:
+      {{- if .Values.ingress.acm_certificate }}
```

Comment:
I would recommend we remove this if condition and make an ACM certificate required.
I don't feel super comfortable exposing this dashboard over an unencrypted connection given the data and permissions it exposes.